src/v0/destinations/google_adwords_enhanced_conversions/networkHandler.js

```diff
@@ -107,7 +107,21 @@ const responseHandler = (destinationResponse) => {
   const message = 'Request Processed Successfully';
   const { status } = destinationResponse;
   if (isHttpStatusSuccess(status)) {
-    // Mostly any error will not have a status of 2xx
+    // for google ads enhance conversions the partialFailureError returns with status 200
+    const { partialFailureError } = destinationResponse.response;
+    // non-zero code signifies partialFailure
+    // Ref - https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto
+    if (partialFailureError && partialFailureError.code !== 0) {
+      throw new NetworkError(
+        `[Google Ads Offline Conversions]:: partialFailureError - ${partialFailureError?.message}`,
```

Comment:
We should return the entire response in stringified form apart from the message only.

Reply:
updated